Jellyfin.Plugin.Bookshelf/Providers/ComicBookImageProvider.cs

```diff
@@ -43,7 +43,7 @@ public async Task<DynamicImageResponse> GetImage(BaseItem item, ImageType type,
             var extension = Path.GetExtension(item.Path);
             if (string.Equals(extension, CbzFileExtension, StringComparison.OrdinalIgnoreCase))
             {
-                return await LoadCover(item);
+                return await LoadCover(item).ConfigureAwait(false);
```

Comment:
Don't await and just return the task? (and remove async from the function signature and wrap the other return in a task)